website/docs/general/community.md

```diff
@@ -8,16 +8,26 @@ keywords:
 description: This article provides useful information about Apache APISIX's Community, including 2 ways to join the Slack channel.
 ---
 
-The Apache APISIX community communicates over Slack and a mailing list. Join both to be in the loop for discussions related to the project and the community.
 
-## Joining the Mailing List
+Apache APISIX is a project maintained by community members under the Apache Software Foundation.
+It is an open source project that abides by the [Apache Software Foundation's code of conduct](https://www.apache.org/foundation/policies/conduct.html). Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API Gateway. It handles traditional north-south traffic, as well as east-west traffic between services.
 
-See the [Subscription Guide](/docs/general/subscribe-guide) to learn more about how you can subscribe to the Apache APISIX mailing list.
+## How to get involved
 
-## Joining the Slack Channel
+Apache APISIX is an open source infrastructure software project that provides API processing and analytics with products and solutions for microservices and real-time traffic processing; such as API gateway, k8s ingress controller, and Service Mesh.
```

Comment:
Doesn't the first sentence already cover this?

Reply:
this is a lot different from the first sentence. Its also a good transition into getting involved